lib/white_bread/outputers/html/formatter.ex

```diff
@@ -0,0 +1,35 @@
+defmodule WhiteBread.Outputers.HTML.Formatter do
+  @moduledoc "HTML formatter"
+
+  def document(content) when is_binary(content) do
+    "<!DOCTYPE html><html>#{content}</html>"
+  end
+
+  def body(content) when is_binary(content) do
+    "<body>#{content}</body>"
+  end
+
+  def list([]) do
+    "Nothing to report."
+  end
+
+  def list(elements) when is_list(elements) and length(elements) > 0 do
```

Comment:
This head could be more concise: `def list([_|_] = elements), do: list(elements,"")`